client/components/add-new-site/style.scss

```diff
@@ -0,0 +1,166 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.add-new-site__button {
+	.gridicon {
+		margin-left: 6px;
+		transition: transform 100ms ease-in-out;
+		height: 18px;
+		width: 18px;
+
+		&.reverse {
+			transform: rotate(180deg);
+		}
+
+		&.mobile {
+			margin-left: 0;
+		}
+	}
+}
+
+.popover.add-new-site__popover {
+	animation: fadeIn 0.2s ease-out;
+
+	@include breakpoint-deprecated(">660px") {
+		margin-inline-start: 32px;
+	}
+
+	.popover__inner {
+		text-align: unset
+	}
+}
+
+.add-new-site__popover-content {
+	display: flex;
+	flex-direction: column;
+	padding: 28px 32px;
+	gap: 32px;
+
+	@include break-large {
+		flex-direction: row;
+	}
+
+	.add-new-site__popover-column {
+		width: 260px;
+		display: flex;
+		flex-direction: column;
+		gap: 8px;
+	}
+}
+
+.add-new-site__popover-column-heading {
+	color: var(--color-accent-40);
+	font-size: rem(12px);
+	font-weight: 500;
+}
+
+button, a {
+	&.components-button.add-new-site__popover-button {
+		padding: 8px;
+		white-space: unset;
+		text-align: unset;
+		margin-block: 0;
+		width: 100%;
+		border-radius: 4px;
+		height: auto;
+		color: var(--color-text-black);
+
+		&:hover {
+			color: var(--color-text-black);
+			background-color: var(--color-neutral-0);
+		}
+
+		&.is-disabled {
+			cursor: not-allowed;
+			background-color: var(--color-neutral-0);
+		}
+
+		@include break-large {
+			display: flex;
+			gap: 8px;
+			align-items: unset;
+			justify-content: unset;
+		}
+
+		&.is-banner {
+			flex-direction: column;
+			gap: 4px;
+			background-color: var(--color-neutral-0);
+
+			@include break-large {
+				padding: 16px;
+				padding-top: 0;
+			}
+
+			.add-new-site__popover-button-icon {
+				height: 78px;
+
+				.sidebar__menu-icon {
+					width: 228px;
+					height: auto;
+				}
+			}
+
+			.add-new-site__popover-button-content {
+				@include break-large {
+					padding: 8px 8px 0 8px;
+				}
+			}
+
+			.add-new-site__popover-button-heading {
+				@include a4a-font-body-md($font-weight: 500);
```

Comment:
I'm unsure if we should reference A4A-specific mixins in the shared scope. 🤔

Reply:
Good catch! Fixed it [here](https://github.com/Automattic/wp-calypso/pull/97499/commits/8365a989faeeda3c1d128f37813e2595c66f08de).